services/galley/journaler/src/Journal.hs

```diff
@@ -44,21 +44,23 @@ runCommand l env c start = void $ C.runClient c $ do
 
     journal :: Row -> C.Client ()
     journal (tid, _, s, time) = C.runClient c $ case s of
-          Just PendingDelete -> liftIO $ journalTeamDelete tid
-          Just Deleted       -> liftIO $ journalTeamDelete tid
-          _                  -> do
-              mems <- Data.teamMembers tid
-              liftIO $ journalTeamCreate tid mems time
+          Just Active        -> journalTeamCreate tid time
+          Just PendingDelete -> journalTeamDelete tid
+          Just Deleted       -> journalTeamDelete tid
+          Just Suspended     -> journalTeamCreate tid time
```

Comment:
I wonder if we should introduce a different event for `suspended` teams?

Reply:
I thought so too when doing this PR. @dkovacevic , wdyt about adding a `TEAM_SUSPEND` [here](https://github.com/wireapp/wire-server/blob/develop/libs/types-common-journal/proto/TeamEvents.proto#L17-L21) ?
Also, if we do it then we should a `TEAM_SUSPEND | TEAM_ACTIVE` event [here](https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/API/Teams.hs#L113-L120) as well.